playbooks/councilmatic/councilmatic.yml

```diff
@@ -14,6 +14,36 @@
       become: true
       raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
 
+    - name: Create login users
+      become: true
+      user:
+        groups: sudo
+        append: true
+        name: '{{ item }}'
+      with_items:
+        - tdooner
+        - howard
+
+    - name: Create authorized keys
+      become: true
+      authorized_key:
+        key: "{{ lookup('file', '../../ssh-keys/' + item + '.pub') }}"
+        user: '{{ item }}'
+      with_items:
+        - tdooner
+        - howard
+
+    - name: Enable sudo for users
+      become: true
+      copy:
+        dest: '/etc/sudoers.d/{{ item }}'
+        content: "# Managed by Ansible\n{{ item }} ALL=(ALL) NOPASSWD:ALL"
```

Comment:
FYI you can use yaml's block syntax to avoid having to encode things like newlines:

```
content: |
  # Managed by Ansible
  {{ item }} ALL=(ALL) NOPASSWD:ALL